src/api/apiClients/axiosApiClient.ts

```diff
@@ -0,0 +1,48 @@
+import axios, { AxiosRequestConfig } from "axios";
+import { BaseApiClient } from "./baseApiClient.js";
+import FormData from "form-data";
+import ReporterService from "../../utils/reporter/reporter.js";
+import LoggerService from "../../utils/logger/logger.js";
+
+class AxiosApiClient extends BaseApiClient {
+  protected createRequestInstance() {
+    this.request = axios.create()
+  }
+
+  protected async send() {
+    return await this.request(this.options as AxiosRequestConfig)
```

Comment:
'return await' outside try/catch block is not used

Reply:
fixed